menu/lxqt-applications-compact.menu

```diff
@@ -203,6 +181,30 @@
 				<Merge type="files"/>
 			</Layout>
 		</Menu>
+	<!-- Other Settings -->
+	<Menu>
+		<Name>DesktopSettings</Name>
+		<Directory>lxqt-settings-other.directory</Directory>
```

Comment:
`lxqt-settings-other.directory` is from lxqt-config. You may need to move it from lxqt-config to lxqt-menu-data.

Reply:
The issue is probably elsewhere, somehow the simple "Preferences" got lost, it's less evident in `lxqt-applications-menu`  but there's twice `lxqt-settings`:

![screen_area_dom_08:16:44_](https://user-images.githubusercontent.com/10681413/226160142-a8bccc31-a2cc-448a-921f-0e3570f52f10.png)

Reply:
Ok, got it: `lxde-settings.directory` was "Preferences", so we could 
* rename it as "lxqt-preferences-directory" and use it in the menus.
*  Or we stick to "Other settings" in all menus.

Reply:
And we have 2 files providing "LXQt Settings" now:` lxqt-settings.directory` provided here and
`lxqt-settings-lxqt.directory` from lxqt-config, the second of better quality but less translations.

Reply:
> Ok, got it: lxde-settings.directory was "Preferences"

Nice catch! Switching to lxqt-preferences.directory sounds good.

> And we have 2 files providing "LXQt Settings" now: lxqt-settings.directory provided here and lxqt-settings-lxqt.directory

You mean https://github.com/lxqt/lxqt-config/blob/master/src/menuname/desktop-directories/lxqt-settings-lxqt.directory.in and https://github.com/lxqt/lxqt-panel/blob/master/menu/desktop-directories/lxqt-settings.directory.in ? Is there a difference in quality?

Reply:
Oh, there are even more around, so there are 3 versions, 2 of `lxqt-settings.directory` counting the one hree from `lxqt-menu-data` which is probablya a result of search+replace of `lxde-settings.directory` done be me time ago. Will look at all later.

Reply:
Using `lxqt-settings-other.directory` would fit better in the `lxqt-applications-compact.menu` but moving it out of `lxqt-config` or creating new dependencies isn't really an option IMO.

Reply:
Why not moving lxqt-applications-compact.menu out of lxqt-config?

Reply:
> Why not moving lxqt-applications-compact.menu out of lxqt-config?

It's in `lxqt-panel` whit the other 2 menus.

Reply:
Oops, my question should be:

Why not moving _lxqt-settings-other.directory_ out of lxqt-config?

Reply:
> Oops, my question should be:
> 
> Why not moving _lxqt-settings-other.directory_ out of lxqt-config?

It's needed by `lxqt-config` , bottom section. If we move it out we would have to make lxqt-config on lxqt-menu-data just for one file.

Reply:
Got it. If necessary, it's acceptable to make lxqt-config depend on lxqt-menu-data. The latter is a tiny package and does not bring other dependencies.